### PR TITLE
Fix unknown error

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
ref: #1394

### CODE REVIEW

File: src/com/owncloud/android/authentication/AuthenticatorActivity.java [hunk 1]
```diff
@@ -1605,13 +1607,18 @@ private void showServerStatus() {
...
-            mServerStatusView.setText(mServerStatusText);
+            setStatusUrl();
             mServerStatusView.setCompoundDrawablesWithIntrinsicBounds(mServerStatusIcon, 0, 0, 0);
             mServerStatusView.setVisibility(View.VISIBLE);
         }
 
     }
 
+    private void setStatusUrl(){
+        String text = getString(R.string.auth_unknown_error_title) + "<br/>" + mHostUrlInput.getText().toString().trim() + "/status.php";
```
Comment (przybylski) [inline src/com/owncloud/android/authentication/AuthenticatorActivity.java]: I think that you should use formatted string:
```java
String text = getString(R.string.auth_unknown_error_title, mHostUrlInput.getText().toString().trim() + "/status.php");
```
and change auth_unknown_error_title string accordingly

**Comment by przybylski** [comment]
+1

**Comment by tobiasKaminsky** [comment]
@davivel this was one of the reason to do this pr: https://github.com/owncloud/android/issues/878

**Comment by danimo** [comment]
@davivel Can we merge this?
@tobiasKaminsky Can you rebase?

**Comment by davivel** [comment]
Sorry, but as I explained in https://github.com/owncloud/android/issues/1394#issuecomment-215497002 , this is bad UX. 

In #878 the request is showing the error message of status.php. Let's do that, not something halfway.

:-1:

**Comment by jesmrec** [comment]
Not planned

**Pull request closed without merging**